ground/gcs/src/plugins/uploader/tl_dfu.cpp

```diff
@@ -469,11 +472,19 @@ bool DFUObject::OpenBootloaderComs(USBPortInfo port)
         AbortOperation();
         if(!EnterDFU()) {
             TL_DFU_QXTLOG_DEBUG(QString("Could not process enterDFU command"));
+            if ((retries--) > 0) {
```

Comment:
Both flight and GCS style call for it...
```
Do not unnecessarily use braces where a single statement will do.

	if (condition)
		action();

and

	if (condition)
		do_this();
	else
		do_that();
```

If you really wanna do it I guess we can.